samza-azure/src/main/java/org/apache/samza/system/eventhub/SamzaEventHubClientManager.java

```diff
@@ -52,29 +54,36 @@
   private final String sasKey;
   private final RetryPolicy retryPolicy;
 
-  public SamzaEventHubClientManager(String eventHubNamespace, String entityPath, String sasKeyName, String sasKey) {
+  public SamzaEventHubClientManager(String eventHubNamespace, String entityPath, String sasKeyName, String sasKey,
+      Integer numClientThreads) {
     this(eventHubNamespace, entityPath, sasKeyName, sasKey,
-            new RetryExponential(MIN_RETRY_BACKOFF, MAX_RETRY_BACKOFF, MAX_RETRY_COUNT, SAMZA_EVENTHUB_RETRY));
+            new RetryExponential(MIN_RETRY_BACKOFF, MAX_RETRY_BACKOFF, MAX_RETRY_COUNT, SAMZA_EVENTHUB_RETRY), numClientThreads);
```

Comment:
Minor: Maybe document what the client threads are used for?

Reply:
Added the documentation in the header.